psi4/driver/procrouting/solvent/pol_embed.py

```diff
@@ -0,0 +1,153 @@
+#
+# @BEGIN LICENSE
+#
+# Psi4: an open-source quantum chemistry software package
+#
+# Copyright (c) 2007-2019 The Psi4 Developers.
+#
+# The copyrights for code used from other parties are included in
+# the corresponding files.
+#
+# This file is part of Psi4.
+#
+# Psi4 is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, version 3.
+#
+# Psi4 is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with Psi4; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# @END LICENSE
+#
+
+import numpy as np
+import cppe
+from qcelemental import constants
+
+from psi4 import core
+from psi4.driver.p4util.exceptions import ValidationError
+
+
+def get_pe_options():
+    if core.get_option('SCF', 'PCM'):
+        raise ValidationError("""Error: 3-layer QM/MM/PCM not implemented.\n""")
+    potfile_name = core.get_local_option('PE', 'POTFILE')
+    pol_embed_options = cppe.PeOptions()
+    pol_embed_options.potfile = potfile_name
+    pol_embed_options.induced_thresh = core.get_local_option('PE', 'INDUCED_CONVERGENCE')
+    pol_embed_options.iso_pol = core.get_local_option('PE', 'ISOTROPIC_POL')
+
+    pol_embed_options.do_diis = core.get_local_option('PE', 'DIIS')
+    pol_embed_options.maxiter = core.get_local_option('PE', 'MAXITER')
+    pol_embed_options.pe_border = core.get_local_option('PE', 'BORDER')
```

Comment:
I think you want `get_option` (same args) in these lines and below rather than `get_local_option`. Details why upon request, but you're better off not knowing. Any tests that previously worked with `set cppe {...}` should now work the same with `set {...}` (provided the option doesn't overlap and contradict a setting in a different used module). It's just a little more flexible if users aren't required to specify the local scope.

If you have experience that contradicts this, let me know.

Reply:
I'd prefer to have the separate input section. In this manner, the input is similar to the other solvent models, e.g., PCM.

Reply:
A `set pe { ... }` block will still work as expected if switch to `get_option`. Right now, `set potfile pna_6w.pot` is going to do nothing, and I worry that will be confusing. But if you really want to try it, ok.

Reply:
Done.